src/main.c

```diff
@@ -730,19 +733,66 @@ static h2o_iovec_t *build_http2_origin_frame(h2o_configurator_command_t *cmd, yo
     return http2_origin_frame;
 }
 
+static ptls_cipher_suite_t **parse_tls13_ciphers(h2o_configurator_command_t *cmd, yoml_t *node, const char *tls13_ciphers)
+{
+    char *p = alloca(strlen(tls13_ciphers) + 1), *saveptr = NULL;
+    strcpy(p, tls13_ciphers);
+    struct {
+        const char *name;
+        ptls_cipher_suite_t *cipher;
+    } cipher_suites[] = {
+        { "TLS_AES_128_GCM_SHA256", &ptls_openssl_aes128gcmsha256},
```

Comment:
How about making the following changes:
* Instead of using `const char *` to hold the names, use `h2o_iovec_t`. The names can be stored like `{H2O_STRLIT("TLS_AES_128_GCM_SHA256")}`.
* Instead of creating a copied string, use `strchr(const_str, ':')` to split the list of cipher-suites being named, then use `h2o_memis` to compare them.